core/tracepoint/event_spec.rb

```diff
@@ -0,0 +1,11 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'TracePoint#event' do
+    it 'returns the type of event' do
+      event_name = nil
+      TracePoint.new(:line) { |tp| event_name = tp.event }.enable
```

Comment:
This test would probably be a good one to show that the block in TracePoint is called multiple times by having the trace for more than one type of event and triggering a couple of events.

Reply:
okay, I have updated the PR.